op-chain-ops/deployer/deployer.go

```diff
@@ -42,19 +44,38 @@ type Deployer func(*backends.SimulatedBackend, *bind.TransactOpts, Constructor)
 
 // NewL1Backend returns a SimulatedBackend suitable for L1. It has
 // the latest L1 hardforks enabled.
-func NewL1Backend() *backends.SimulatedBackend {
-	return NewBackendWithGenesisTimestamp(0, true)
+func NewL1Backend() (*backends.SimulatedBackend, error) {
+	backend, err := NewBackendWithGenesisTimestamp(ChainID, 0, true, nil)
```

Comment:
There is no real need to check for an error here - notice its just being passed thru. It would only be useful to do this if you annotated the error like so `return nil, fmt.Errorf("cannot create L1 backend: %w, err")`